semantics/Makefile

```diff
@@ -25,6 +25,10 @@ K_INCLUDE_DIRS += $(PROFILE_DIR)/semantics/cpp
 # Appending to whatever the environment provided.
 KDEP_FLAGS += $(addprefix -I ,$(K_INCLUDE_DIRS))
 KOMPILE_FLAGS += $(addprefix -I ,$(K_INCLUDE_DIRS))
+KOMPILE_FLAGS += --no-prelude
+KOMPILE_FLAGS += -w all
+KOMPILE_FLAGS += -v
+KOMPILE_FLAGS += --debug
```

Comment:
These flags were all used by all timestamp targets, with the exception of c-nd-thread. Now it too will use the same set of flags.